helm/nessie/values.yaml

```diff
@@ -1,5 +1,5 @@
 # -- The number of replicas to deploy (horizontal scaling).
-# Beware that replicas are stateless; don't set this number > 1 when using INMEMORY or ROCKS version store types.
+# Beware that replicas are stateless; don't set this number > 1 when using IN_MEMORY or ROCKSDB version store types.
```

Comment:
Line 31 and many places in this file still uses the old model names. Do we need to replace them too?

Reply:
Good catch, I will do a more thorough pass on the file.